tests/homework1/test_task03.py

```diff
@@ -0,0 +1,24 @@
+import os
+from typing import Tuple
+
+import pytest
+
+from homework1.task03 import find_maximum_and_minimum
+
+path = "tests/homework1/input_files"
```

Comment:
It's better to call the variable more specific not to collapse with os.path name. For example, `test_files_path`